tests/gen_cert_test.py

```diff
@@ -150,12 +151,33 @@ def test_render_flair_function_true():
     assert_true(draw_flair(cert, flair, 'top', page, context=None))
 
 
-def test_cert_date_timezone():
-    """
-    Make sure certs render dates according to the passed timezone
-    """
-    today_date = datetime.datetime(2016, 8, 22, 0, 0, 0, 0)
-    utc_cert_date = get_cert_date(today_date, timezone='UTC')
-    assert_true(utc_cert_date == 'August 22, 2016')
-    pacific_cert_date = get_cert_date(today_date, timezone='US/Pacific')
-    assert_true(pacific_cert_date == 'August 21, 2016')
+class CertificateTests_Dates():
+    """Tests for the certificates code"""
+
+    def setUp(self):
+        datetime_patcher = mock.patch.object(
+            my_module.datetime, 'datetime',
+            mock.Mock(wraps=datetime.datetime)
+        )
+        mocked_datetime = datetime_patcher.start()
+        mocked_datetime.today.return_value = datetime.strptime('2016-08-22 04:00:00', '%Y-%m-%d %H:%M:%S')
+        self.addCleanup(datetime_patcher.stop)
+
+    def test_cert_date_timezone():
```

Comment:
We should also have a test case that ensures that this all still works if you don't pass in a timezone at all.